plsql/PlSqlParser.g4

```diff
@@ -2071,13 +2077,14 @@ log_grp
     ;
 
 supplemental_table_logging
-    : ADD
-       (','? SUPPLEMENTAL LOG  (supplemental_log_grp_clause | supplemental_id_key_clause) )*
-    | DROP (','? SUPPLEMENTAL LOG (supplemental_id_key_clause | GROUP log_grp) )*
+    : ADD (SUPPLEMENTAL LOG  (supplemental_log_grp_clause | supplemental_id_key_clause) )?
```

Comment:
Are you sure that `?` is required here?

Reply:
Yes, `(','? xxx)*` should be changed to `xxx? (',' xxx)*`

Reply:
But now i see there is a mistake with the former grammar, it should be `(','? xxx)+`. That `?` will be removed.

Reply:
I think it's better to replace `(','? xxx)*` with `(xxx (',' xxx)*)?`